docs/developer/discounts/vouchers.mdx

```diff
@@ -24,6 +24,7 @@ We can also distinguish vouchers by their value type:
 - `PERCENTAGE`: reduces the price by a specified percentage value.
 
 By setting `Voucher.applyOncePerOrder`, there is an option to apply the discount only to the cheapest eligible product.
+The discount reduces the value of a single unit, not the entire line.
```

Comment:
```suggestion
The discount reduces the price of a single quantity of the cheapest product.
```
Maybe something like this ? ☝️ I don't think that we need to explicitly say, that this voucher is not applied on entire line. I mean, this is already clear from first part of the sentence.
I've also changed the unit into quantity, as we are using quantity in the OrderLine, CheckoutLine, so I assumed that this would better fit here. 
But this is only the suggestions 😄

Reply:
Thank you for the suggestion. The purpose of this sentence is to emphasize that the this voucher is not applied to the entire line (like the all other line-level discounts), so I would keep it. I used "single unit" as I am not sure if "single quantity" phrase exists in english. Please note, that the info, that it is applied to the cheapest product is contained in the sentence right before. This my rationale, let's wait for other reviewers and let them decide what fits here better.

Reply:
Maybe:
```The discount applies to a single unit of the product, not the entire order line.`